services/provider/server/consumer.go

```diff
@@ -227,3 +227,16 @@ func (c *ocsConsumerManager) GetByClientID(ctx context.Context, clientID string)
 	}
 	return nil, nil
 }
+
+func (c *ocsConsumerManager) ClearClientInformation(ctx context.Context, consumerID string) error {
+	storageConsumer, err := c.Get(ctx, consumerID)
+	if err != nil {
+		return fmt.Errorf("failed to get storageConsumer resource with the provided UUID. %v", err)
+	}
+	storageConsumer.Status.Client = nil
+	err = c.client.Status().Update(ctx, storageConsumer)
+	if err != nil {
+		return fmt.Errorf("failed to remove client information from storageConsumer status %v: %v", storageConsumer.Name, err)
+	}
```

Comment:
The server does not use a cached client, this mean that the get goes directly into the API server. Can we use a patch instead to remove the need for the get operation?